values-secret.yaml.template

```diff
@@ -3,4 +3,16 @@ secrets:
   # NEVER COMMIT THESE VALUES TO GIT
   config-demo:
     # Secret used for demonstrating vault storage, external secrets, and ACM distribution
-    secret: PLAINTEXT
+    config-demo-secret: VALUE
+
+  # Required for automated spoke deployment
+  aws:
+      access_key_id: VALUE
+      secret_access_key: VALUE
+
+# Required for automated spoke deployment
+# DO NOT use ~/ , it will appear to work be the secrets will be empty
```

Comment:
This has actually been fixed now via https://github.com/hybrid-cloud-patterns/common/pull/138